http_client.go

```diff
@@ -172,13 +174,16 @@ func (c *httpClient) post(ctx context.Context, message proto.Message) (proto.Mes
 	}
 
 	req.Header.Set("Content-Type", "application/x-google-protobuf")
+	for k, v := range c.authConfig.additionalHeaders {
```

Comment:
This should be removed since we want to allow the user to pass in a `http.Client` when using the connector.

Reply:
done